src/helper.cc

```diff
@@ -931,10 +927,10 @@ helper_stateful_server::HelperServerClosed(helper_stateful_server *srv)
 }
 
 Helper::Xaction *
-helper_server::popRequest(int request_number)
+Helper::Session::popRequest(const int request_number)
 {
-    Helper::Xaction *r = nullptr;
-    helper_server::RequestIndex::iterator it;
+    Xaction *r = nullptr;
+    RequestIndex::iterator it;
     if (parent->childs.concurrency) {
         // If concurrency supported retrieve request from ID
         it = requestsIndex.find(request_number);
```

Comment:
The iterator can be better scoped and probably use `auto` as well.
```suggestion
    if (parent->childs.concurrency) {
        // If concurrency supported retrieve request from ID
        const auto it = requestsIndex.find(request_number);
```